lib/rails_heroicon/helper.rb

```diff
@@ -32,9 +32,22 @@ module Helper
     # == Accessibility
     # The helper method automatically sets <tt>aria-hidden=true</tt> if <tt>aria-label</tt> is not set, and
     # if <tt>aria-label</tt> is set, then <tt>role=img</tt> is set automatically.
+    #
+    # You can provide tooltips on hover if you pass in an `:alt` option. Anything passed into the `:alt` option
+    # will be rendered inside of a <title> tag within the rendered SVG.
     def heroicon(symbol, **options)
       icon = RailsHeroicon.new(symbol, **options)
-      content_tag(:svg, icon.svg_path.html_safe, icon.options)
+
+      if alt = options.delete(:alt)
+        content_tag(:svg, icon.options) do
+          [
+            content_tag(:title, alt),
+            icon.svg_path
+          ].join.html_safe
+        end
+      else
+        content_tag(:svg, icon.svg_path.html_safe, icon.options)
+      end
```

Comment:
Let's go with `title` tag since we already have `aria-label` support.

```ruby
def heroicon(symbol, title: nil, **options)
  icon = RailsHeroicon.new(symbol, **options)

  title_tag = content_tag(:title, title) if title
  content_tag(:svg, title_tag.to_s.html_safe + icon.svg_path.html_safe, icon.options)
end
```

Let's also add some tests.
```ruby
# spec/spec_helper.rb

require "rails_heroicon/helper"
```

```ruby
# spec/rails_heroicon/helper_spec.rb

RSpec.describe RailsHeroicon::Helper do
  before do
    helper = Class.new do
      include RailsHeroicon::Helper
      include ActionView::Helpers::TagHelper
    end

    stub_const("Heroicon", helper)
  end

  describe "#heroicon" do
    context "when title is specified" do
      it "renders the title tag" do
        icon = Heroicon.new.heroicon("user", title: "My title")

        expect(icon).to match(/<title>My title<\/title>/)
      end
    end

    context "when title is not specified" do
      it "does not render the title tag" do
        icon = Heroicon.new.heroicon("user")

        expect(icon).not_to match(/<title>My title<\/title>/)
      end
    end
  end
end
```

Reply:
Thanks for the tips, I've pushed an update with your suggested changes (and updated the README with documentation)

Reply:
Thank you for your contribution! 🥳